src/common/components/forms/AcceptOfferForm.tsx

```diff
@@ -63,9 +69,15 @@ const AcceptOfferForm: FC<AcceptOfferFormProps> = ({
   const onAcceptOffer = async () => {
     if (offer) {
       if (listing) {
-        await sdk.offers(marketplace.auctionHouse).accept({ offer, nft, cancel: [listing] });
+        await sdk
+          .transaction()
+          .add(sdk.offers(marketplace.auctionHouses[0]).accept({ offer, nft, cancel: [listing] }))
```

Comment:
Hi Austin, Right now holaplex is going to support just a single spl-token i.e sol. So, only one auctionHouse will be present in the list. We can modify this (as you suggested) as the requirement arises. 

And for listing there can be only one auctionHouse per listing as a particular listing can only belong to a single spl-token. Similarly for offer and purchase.

@kespinola Do you want to add something here?

Reply:
That's correct! Apart from cases where we are creating a new offer/listing/purchase, it would be better to use auctionHouse directly from that particular object. 

And in cases where we are creating a new offer/listing/purchase we can get auctionHouse from marketplace.auctionHouses, example:

```
await sdk
        .transaction()
        .add(
          sdk
            .listings(
              marketplace.auctionHouses.filter(
                (ah) => ah.treasuryMint === selectedToken?.address
              )[0]
            )
            .post({
              amount,
              nft,
            })
        )
        .send()
```